src/components/document/resize-panel-divider.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import DragThumbnailIcon from "../../assets/drag-thumb-icon.svg";
+import LeftRightDragIcon from "../../assets/left-right-drag.svg";
+import "./resize-panel-divider.scss";
+
+interface IProps {
+  isResourceExpanded: boolean,
+  dividerPosition: number,
```

Comment:
I think you want semicolons after these.

Reply:
It has always struck me as odd that `typescript-eslint` doesn't check that for interfaces.

Reply:
Update: a quick google search suggests that there is a `typescript-eslint` rule for that but that we're still relying on `eslint`'s semicolon rule, so we'd have to change our eslint config to have TypeScript-only constructs linted for semicolons.